### PR TITLE
Adds commonly used mixins and variables

### PR DESCRIPTION
**Pull request by helrond**
Adds commonly used mixins and variables:
- responsive mixins
- transition mixin
- border-radius variable

fixes #76

### CODE REVIEW

**Comment by HaSistrunk** [comment]
We are taking a mobile-first approach, right? So conceptually we are writing styles for small screens, and then using the `md-up` and `lg-up` mixins to change those styles as the screen gets larger? I'm wondering if it would make sense to document that approach in comments along with these mixins?

**Comment by helrond** [comment]
@HaSistrunk that's right - we're taking a mobile-first approach. I can add to the comments.

File: stylesheets/abstracts/_mixins.scss [hunk 1]
```diff
@@ -37,3 +37,99 @@
...
+@mixin transition-default {
+  @media screen and (prefers-reduced-motion: no-preference) {
+    transition: all 0.2s ease;
+  }
+}
+
+/**
+ * Hide element while making it readable for screen readers
+ * Shamelessly borrowed from HTML5Boilerplate:
+ * https://github.com/h5bp/html5-boilerplate/blob/master/src/css/main.css#L119-L133
```
Comment (HaSistrunk) [inline stylesheets/abstracts/_mixins.scss]:
```suggestion
 * https://github.com/h5bp/html5-boilerplate/blob/e460abb30cf2fef924c64ec1f7de8e3f8c23452a/dist/css/style.css#L103-L123
```

**Review by HaSistrunk — APPROVED** [review]
Great!

**Pull request merged**